pkg/cli/environment.go

```diff
@@ -91,5 +101,6 @@ func (s *EnvSettings) EnvVars() map[string]string {
 		"HELM_REGISTRY_CONFIG":   s.RegistryConfig,
 		"HELM_REPOSITORY_CACHE":  s.RepositoryCache,
 		"HELM_REPOSITORY_CONFIG": s.RepositoryConfig,
+		"HELM_NAMESPACE":         s.Namespace(),
```

Comment:
Trying to adopt the helm-diff plugin for helm3 it came up that plugins are unaware of  `--kube-context` and `--namespace` passed to the main binary. This makes it impossible to talk to the k8s cluster used by helm:
https://github.com/databus23/helm-diff/pull/151#issuecomment-537679821

Not sure if this is the right place but how about passing the values from `--kube0context` as `HELM_KUBECONTEXT`?

Reply:
Also of note from that issue in helm-diff is that @databus23 pointed out that we still have `--kube-context` as a top level flag in the helm CLI, and now we will also have `--context` due to the flags we attach with the `AddFlags` method

Reply:
A dedicated issue opened for adding those envvars: https://github.com/helm/helm/issues/6631